Enemies/Enemy.gd

```diff
@@ -246,18 +271,22 @@ func get_path_distance(path_array: Array) -> float:
 	return total_distance
 
 
+
 # Respond to player attacks
-var HP: float = 2.0
+var HP: float = 0.0
 
 
 func take_damage(damage: float) -> void:
 	world_state["has_target"] = true
 
 	alert_comrades()
-	HP -= damage
-	if HP > 0:
-		$CSGCombiner/CSGCylinder.visible = false
-		$CSGCombiner/CSGCylinder2.visible = true
+	$CSGCombiner.get_node("CSGCylinder" + str(int(HP))).visible = false
+	HP += damage
```

Comment:
This method of HP counting up may be a little confusing to work with if there are other parts of the code that deal with HP. I think this would be better if the HP variable was renamed to "damage_taken" or if the old way of subtracting HP was kept and the in_danger calculation was modified.

Reply:
changed to damage_taken